pii_recognition/registers/recogniser_registry_test.py

```diff
@@ -0,0 +1,8 @@
+from .recogniser_registry import RecogniserRegistry
+
+
+def test_RecogniserRegistry():
+    actual = RecogniserRegistry()
+    assert isinstance(actual, dict)
+    assert len(actual.keys()) > 0
```

Comment:
why not test the exact key length?

Reply:
I thought about that and the consideration is that as the project moves, we may have more and more predefined recognisers, regex recognisers. Whenever adding a recogniser, test has to be updated accordingly, a lot of manual changes involved.

Here the test is to check if the registry is empty and my assumption is that if we want to use a registry, there must be something in it. So far, this assumption is still true. And if you have any other thoughts on more efficient implementation of registry as well as testing, drop a line or two.